app/screens/channel/channel_nav_bar/channel_drawer_button.js

```diff
@@ -133,8 +132,10 @@ class ChannelDrawerButton extends PureComponent {
                 style={style.container}
             >
                 <View style={[style.wrapper, {opacity: this.state.opacity}]}>
-                    {icon}
-                    {badge}
+                    <View>
```

Comment:
is this `<View>` needed?

Reply:
Yup the view is needed to wrap the styles for `position absolute` other wise they move based on the available space hence the issue